app/views/items/show.html.erb

```diff
@@ -27,6 +27,43 @@
   </script>
   <%= render "item_largescreen" %>
   <%= render "item_smallscreen" %>
+
+  <%# lending history of this item %>
+  <% if current_user.can_manage?(@item) %>
+    <h2><%= t('analytics.history.headline') %></h2>
+    <% if @item.lending_history.length <= 0 %>
+      <div class="d-flex justify-content-center">
+        <h4><%= t('analytics.history.empty') %></h4>
+      </div>
+    <% else %>
+    <div class="table-responsive">
+      <table class="table table-hover">
+        <thead>
+          <tr>
+            <th scope="col">User</th>
+            <th scope="col">Date borrowed</th>
+            <th scope="col">Date returned</th>
+          </tr>
```

Comment:
please add internalization for columns. Also would recommend renaming "User" to "Borrowed by"